src/components/pages/register.js

```diff
@@ -119,7 +119,9 @@ class Register extends React.Component {
                   <a
                     key="terms-link"
                     title={settings.termsPage?.[language]?.linkTitle || null}
-                    href={settings.termsPage?.[language]?.documentSlug ? urls.getArticleUrl(settings.termsPage[language].documentSlug) : '#'}
+                    href={settings.termsPage?.[language]?.documentKey
+                      ? urls.getDocUrl(settings.termsPage[language].documentKey, settings.helpPage[language].documentSlug)
+                      : '#'}
```

Comment:
Can you please help me understand this url composition? I don't really get the `settings.helpPage` part 😬